commands/project.go

```diff
@@ -18,10 +18,10 @@
 package commands
 
 import (
-	"github.com/apache/incubator-openwhisk-cli/wski18n"
-	"github.com/apache/incubator-openwhisk-wskdeploy/cmd"
-	"github.com/apache/incubator-openwhisk-wskdeploy/utils"
-	wskdeploy_wski18n "github.com/apache/incubator-openwhisk-wskdeploy/wski18n"
+	"github.com/apache/openwhisk-cli/wski18n"
+	"github.com/apache/openwhisk-wskdeploy/cmd"
```

Comment:
do you need to fix wskdeploy first/at the same time?

Reply:
working on it...